tests/deploy/sa-agent-pytest.yml

```diff
@@ -0,0 +1,740 @@
+            period: 10s
+          - id: system/metrics-system.memory-543f80aa-253c-4589-8bf3-3b0608c1d14b
+            data_stream:
+              type: metrics
+              dataset: system.memory
+            metricsets:
+              - memory
+            period: 10s
+          - id: system/metrics-system.network-543f80aa-253c-4589-8bf3-3b0608c1d14b
+            data_stream:
+              type: metrics
+              dataset: system.network
+            metricsets:
+              - network
+            period: 10s
+            network.interfaces: null
+          - id: system/metrics-system.process-543f80aa-253c-4589-8bf3-3b0608c1d14b
+            data_stream:
+              type: metrics
+              dataset: system.process
+            metricsets:
+              - process
+            period: 10s
+            process.include_top_n.by_cpu: 5
+            process.include_top_n.by_memory: 5
+            process.cmdline.cache.enabled: true
+            process.cgroups.enabled: false
+            process.include_cpu_ticks: false
+            processes:
+              - .*
+          - id: >-
+              system/metrics-system.process.summary-543f80aa-253c-4589-8bf3-3b0608c1d14b
+            data_stream:
+              type: metrics
+              dataset: system.process.summary
+            metricsets:
+              - process_summary
+            period: 10s
+          - id: >-
+              system/metrics-system.socket_summary-543f80aa-253c-4589-8bf3-3b0608c1d14b
+            data_stream:
+              type: metrics
+              dataset: system.socket_summary
+            metricsets:
+              - socket_summary
+            period: 10s
+          - id: system/metrics-system.uptime-543f80aa-253c-4589-8bf3-3b0608c1d14b
+            data_stream:
+              type: metrics
+              dataset: system.uptime
+            metricsets:
+              - uptime
+            period: 10s
+        meta:
+          package:
+            name: system
+            version: 1.19.1
+      - id: 3f8cbb75-38e5-465e-a4aa-ee577d67eccd
+        revision: 2
+        name: sa-agent
+        type: cloudbeat/cis_k8s
+        data_stream:
+          namespace: default
+        use_output: default
+        package_policy_id: 3f8cbb75-38e5-465e-a4aa-ee577d67eccd
+        streams:
+          - id: >-
+              cloudbeat/cis_k8s-cloud_security_posture.findings-3f8cbb75-38e5-465e-a4aa-ee577d67eccd
+            data_stream:
+              type: logs
+              dataset: cloud_security_posture.findings
+            name: Findings
+            processors:
+              - add_cluster_id: null
+            runtime_cfg:
+              activated_rules:
+                cis_k8s:
+                  - cis_1_2_18
+                  - cis_1_2_20
+                  - cis_1_2_19
+                  - cis_1_2_16
+                  - cis_1_2_32
+                  - cis_1_1_15
+                  - cis_4_2_6
+                  - cis_4_1_10
```

Comment:
It might be problematic if we add more rules, wouldn't it?

Reply:
it is temporary solution for integrating standalone agent to current solution.
Since we are planning to manage cloudbeat / agent on environment infrastructure level deploy yamls will be removed from automation tests infra.

Reply:
sounds good